2013-06-23 23:06:39,365 - DEBUG: ===============================================Begin Log===============================================
2013-06-23 23:06:39,367 - DEBUG: 


2013-06-23 23:06:39,367 - DEBUG: batIdx: 0
2013-06-23 23:06:39,368 - DEBUG: numUnvisit: 6, self.numBaobei: 3
2013-06-23 23:06:39,368 - DEBUG: self.randomVisit: [0, 3, 4]
2013-06-23 23:06:39,395 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 302, in <module>
    hasUnvisit = view_3_baobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 245, in view_3_baobei
    viewer.createBaobei(idx)
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 195, in createBaobei
    ieExplorer.openURL(url)
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 130, in openURL
    self.ie = existIE(url)
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 43, in existIE
    ies = getAllRunningIE()
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 18, in getAllRunningIE
    for ie in ies:
  File "C:\Python27\lib\site-packages\win32com\client\util.py", line 84, in next
    return _get_good_object_(self._iter_.next(), resultCLSID = self.resultCLSID)
com_error: (-2147417856, '\xcf\xb5\xcd\xb3\xb5\xf7\xd3\xc3\xca\xa7\xb0\xdc\xa1\xa3', None, None)

2013-06-23 23:06:39,407 - ERROR: closeAllRunningIE exception
2013-06-23 23:06:39,407 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 24, in closeAllRunningIE
    ies = getAllRunningIE()
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 18, in getAllRunningIE
    for ie in ies:
  File "C:\Python27\lib\site-packages\win32com\client\util.py", line 84, in next
    return _get_good_object_(self._iter_.next(), resultCLSID = self.resultCLSID)
com_error: (-2147417856, '\xcf\xb5\xcd\xb3\xb5\xf7\xd3\xc3\xca\xa7\xb0\xdc\xa1\xa3', None, None)

2013-06-23 23:10:25,792 - DEBUG: ===============================================Begin Log===============================================
2013-06-23 23:10:25,793 - DEBUG: 


2013-06-23 23:10:25,793 - DEBUG: batIdx: 0
2013-06-23 23:10:25,795 - DEBUG: numUnvisit: 6, self.numBaobei: 3
2013-06-23 23:10:25,795 - DEBUG: self.randomVisit: [1, 4, 5]
2013-06-23 23:10:25,825 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 302, in <module>
    hasUnvisit = view_3_baobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 245, in view_3_baobei
    viewer.createBaobei(idx)
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 195, in createBaobei
    ieExplorer.openURL(url)
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 130, in openURL
    self.ie = existIE(url)
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 43, in existIE
    ies = getAllRunningIE()
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 18, in getAllRunningIE
    for ie in ies:
  File "C:\Python27\lib\site-packages\win32com\client\util.py", line 84, in next
    return _get_good_object_(self._iter_.next(), resultCLSID = self.resultCLSID)
com_error: (-2147417856, '\xcf\xb5\xcd\xb3\xb5\xf7\xd3\xc3\xca\xa7\xb0\xdc\xa1\xa3', None, None)

2013-06-23 23:10:25,835 - ERROR: closeAllRunningIE exception
2013-06-23 23:10:25,835 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 24, in closeAllRunningIE
    ies = getAllRunningIE()
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 18, in getAllRunningIE
    for ie in ies:
  File "C:\Python27\lib\site-packages\win32com\client\util.py", line 84, in next
    return _get_good_object_(self._iter_.next(), resultCLSID = self.resultCLSID)
com_error: (-2147417856, '\xcf\xb5\xcd\xb3\xb5\xf7\xd3\xc3\xca\xa7\xb0\xdc\xa1\xa3', None, None)

2013-06-23 23:14:39,118 - DEBUG: ===============================================Begin Log===============================================
2013-06-23 23:14:39,164 - DEBUG: 


2013-06-23 23:14:39,164 - DEBUG: batIdx: 0
2013-06-23 23:14:39,180 - DEBUG: numUnvisit: 6, self.numBaobei: 3
2013-06-23 23:14:39,180 - DEBUG: self.randomVisit: [4, 1, 2]
2013-06-23 23:14:45,499 - DEBUG: createBaobei visitIdx(0): http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.13.pBqNrr&id=20371475288&bucket_id=19&scene=taobao_shop

2013-06-23 23:14:48,325 - DEBUG: createBaobei visitIdx(1): http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop

2013-06-23 23:15:07,811 - DEBUG: createBaobei visitIdx(2): http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop

2013-06-23 23:15:07,812 - DEBUG: mainIE waitBusy before setForeground: 0
2013-06-23 23:15:09,868 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 302, in <module>
    hasUnvisit = view_3_baobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 257, in view_3_baobei
    mainIE.setForeground()
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 200, in setForeground
    win32gui.SetForegroundWindow(self.ie.hwnd)
error: (0, 'SetForegroundWindow', 'No error message is available')

2013-06-23 23:15:11,812 - DEBUG: len(ies): 4
2013-06-23 23:15:11,822 - DEBUG: ie.LocationURL: file:///G:/WorkStation/GitAutoVerify
2013-06-23 23:15:11,871 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.13.pBqNrr&id=20371475288&bucket_id=19&scene=taobao_shop
2013-06-23 23:15:11,872 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.13.pBqNrr&id=20371475288&bucket_id=19&scene=taobao_shop
2013-06-23 23:15:12,943 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop
2013-06-23 23:15:12,946 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop
2013-06-23 23:15:14,428 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop
2013-06-23 23:15:14,461 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop
2013-06-23 23:15:56,664 - DEBUG: ===============================================Begin Log===============================================
2013-06-23 23:15:56,664 - DEBUG: 


2013-06-23 23:15:56,664 - DEBUG: batIdx: 0
2013-06-23 23:15:56,664 - DEBUG: numUnvisit: 6, self.numBaobei: 3
2013-06-23 23:15:56,664 - DEBUG: self.randomVisit: [3, 0, 5]
2013-06-23 23:15:58,973 - DEBUG: createBaobei visitIdx(0): http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.9.yj005q&id=17234868278&scene=taobao_shop

2013-06-23 23:16:01,138 - DEBUG: createBaobei visitIdx(1): http://detail.tmall.com/item.htm?id=10000202729&spm=a230r.1.14.9.ff5tiJ&ad_id=&am_id=&cm_id=140105335569ed55e27b&pm_id=

2013-06-23 23:16:05,769 - DEBUG: createBaobei visitIdx(2): http://detail.tmall.com/item.htm?spm=a220o.1000855.w31-18405562885.9.yj005q&id=12935068038
2013-06-23 23:16:05,769 - DEBUG: mainIE waitBusy before setForeground: 0
2013-06-23 23:16:06,536 - DEBUG: openCurBaobei
2013-06-23 23:16:11,525 - DEBUG: subIdx: 0, url: http://detail.tmall.com/item.htm?id=16245152121
2013-06-23 23:16:11,540 - DEBUG: come back to mainIE waitBusy before setForeground: 
2013-06-23 23:16:11,586 - DEBUG: come back to mainIE waitBusy after setForeground: 
2013-06-23 23:16:11,586 - DEBUG: self.mainIE.scrollToNode
2013-06-23 23:17:03,220 - DEBUG: self.createNewSubIE
2013-06-23 23:17:03,937 - DEBUG: subIE.stayInSubPage
2013-06-23 23:17:06,397 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 302, in <module>
    hasUnvisit = view_3_baobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 264, in view_3_baobei
    baobei.openCurBaobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 117, in openCurBaobei
    while subIE.waitBusy(IE_TIME_OUT_NEW_PAGE)==True:
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 179, in waitBusy
    time.sleep(deltaTime)
KeyboardInterrupt

2013-06-23 23:17:07,378 - DEBUG: len(ies): 5
2013-06-23 23:17:07,394 - DEBUG: ie.LocationURL: file:///G:/WorkStation/GitAutoVerify
2013-06-23 23:17:07,642 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.9.yj005q&id=17234868278&scene=taobao_shop
2013-06-23 23:17:07,642 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.9.yj005q&id=17234868278&scene=taobao_shop
2013-06-23 23:17:08,720 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?id=10000202729&spm=a230r.1.14.9.ff5tiJ&ad_id=&am_id=&cm_id=140105335569ed55e27b&pm_id=
2013-06-23 23:17:08,734 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?id=10000202729&spm=a230r.1.14.9.ff5tiJ&ad_id=&am_id=&cm_id=140105335569ed55e27b&pm_id=
2013-06-23 23:17:09,749 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w31-18405562885.9.yj005q&id=12935068038
2013-06-23 23:17:09,749 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w31-18405562885.9.yj005q&id=12935068038
2013-06-23 23:17:10,763 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?id=16245152121
2013-06-23 23:17:10,763 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?id=16245152121
2013-06-23 23:34:25,417 - DEBUG: ===============================================Begin Log===============================================
2013-06-23 23:34:25,447 - DEBUG: 


2013-06-23 23:34:25,447 - DEBUG: batIdx: 0
2013-06-23 23:34:25,470 - DEBUG: numUnvisit: 6, self.numBaobei: 3
2013-06-23 23:34:25,470 - DEBUG: self.randomVisit: [2, 5, 1]
2013-06-23 23:34:26,117 - DEBUG: createBaobei visitIdx(0): http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop

2013-06-23 23:34:26,911 - DEBUG: createBaobei visitIdx(1): http://detail.tmall.com/item.htm?spm=a220o.1000855.w31-18405562885.9.yj005q&id=12935068038
2013-06-23 23:34:35,565 - DEBUG: createBaobei visitIdx(2): http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop

2013-06-23 23:34:35,565 - DEBUG: mainIE waitBusy before setForeground: 0
2013-06-23 23:34:35,878 - DEBUG: type(self.ie.hwnd): <type 'int'>
2013-06-23 23:34:35,878 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 302, in <module>
    hasUnvisit = view_3_baobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 257, in view_3_baobei
    mainIE.setForeground()
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 199, in setForeground
    PyHANDLE = pywintypes.HANDLE(self.ie.hwnd)
NameError: global name 'pywintypes' is not defined

2013-06-23 23:34:36,348 - DEBUG: len(ies): 7
2013-06-23 23:34:36,397 - DEBUG: ie.LocationURL: file:///G:/WorkStation/zhubajie/2897106淘宝店铺刷流量/20130623
2013-06-23 23:34:36,479 - DEBUG: ie.LocationURL: http://www.baidu.com/s?ie=utf-8&bs=bing%E6%90%9C%E7%B4%A2&f=8&rsv_bp=1&rsv_spt=3&wd=google%E6%90%9C%E7%B4%A2&rsv_sug3=2&rsv_sug=0&rsv_sug1=2&rsv_sug4=78&inputT=1538
2013-06-23 23:34:36,482 - DEBUG: closeAllRunningIE: http://www.baidu.com/s?ie=utf-8&bs=bing%E6%90%9C%E7%B4%A2&f=8&rsv_bp=1&rsv_spt=3&wd=google%E6%90%9C%E7%B4%A2&rsv_sug3=2&rsv_sug=0&rsv_sug1=2&rsv_sug4=78&inputT=1538
2013-06-23 23:34:37,493 - DEBUG: ie.LocationURL: http://www.google.com.hk/webhp?hl=zh-CN&sourceid=cnhp#newwindow=1&safe=strict&hl=zh-CN&site=webhp&source=hp&q=TypeError:+The+object+is+not+a+PyHANDLE+object&oq=TypeError:+The+object+is+not+a+PyHANDLE+object&gs_l=hp.12...3491.3491.0.4720.1.1.0.0.0.0.0.0..0.0...0.0...1c.1.17.hp.fokq6eQWTuM&bav=on.2,or.&bvm=bv.48293060,d.dGI&fp=96ae074e703af13&biw=684&bih=422
2013-06-23 23:34:37,496 - DEBUG: closeAllRunningIE: http://www.google.com.hk/webhp?hl=zh-CN&sourceid=cnhp#newwindow=1&safe=strict&hl=zh-CN&site=webhp&source=hp&q=TypeError:+The+object+is+not+a+PyHANDLE+object&oq=TypeError:+The+object+is+not+a+PyHANDLE+object&gs_l=hp.12...3491.3491.0.4720.1.1.0.0.0.0.0.0..0.0...0.0...1c.1.17.hp.fokq6eQWTuM&bav=on.2,or.&bvm=bv.48293060,d.dGI&fp=96ae074e703af13&biw=684&bih=422
2013-06-23 23:34:38,509 - DEBUG: ie.LocationURL: http://stackoverflow.com/questions/3538976/python-pyhandle-object-in-win32gui
2013-06-23 23:34:38,513 - DEBUG: closeAllRunningIE: http://stackoverflow.com/questions/3538976/python-pyhandle-object-in-win32gui
2013-06-23 23:34:39,525 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop
2013-06-23 23:34:39,528 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop
2013-06-23 23:34:40,535 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w31-18405562885.9.yj005q&id=12935068038
2013-06-23 23:34:40,538 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w31-18405562885.9.yj005q&id=12935068038
2013-06-23 23:34:41,562 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop
2013-06-23 23:34:41,562 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop
2013-06-23 23:35:25,947 - DEBUG: ===============================================Begin Log===============================================
2013-06-23 23:35:25,947 - DEBUG: 


2013-06-23 23:35:25,947 - DEBUG: batIdx: 0
2013-06-23 23:35:25,947 - DEBUG: numUnvisit: 6, self.numBaobei: 3
2013-06-23 23:35:25,947 - DEBUG: self.randomVisit: [5, 1, 2]
2013-06-23 23:35:28,430 - DEBUG: createBaobei visitIdx(0): http://detail.tmall.com/item.htm?spm=a220o.1000855.w31-18405562885.9.yj005q&id=12935068038
2013-06-23 23:35:31,891 - DEBUG: createBaobei visitIdx(1): http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop

2013-06-23 23:35:35,983 - DEBUG: createBaobei visitIdx(2): http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop

2013-06-23 23:35:35,983 - DEBUG: mainIE waitBusy before setForeground: 0
2013-06-23 23:35:35,987 - DEBUG: type(self.ie.hwnd): <type 'int'>
2013-06-23 23:35:37,197 - DEBUG: openCurBaobei
2013-06-23 23:35:39,559 - DEBUG: subIdx: 0, url: http://detail.tmall.com/item.htm?id=18404885879
2013-06-23 23:35:39,563 - DEBUG: come back to mainIE waitBusy before setForeground: 
2013-06-23 23:35:39,594 - DEBUG: type(self.ie.hwnd): <type 'int'>
2013-06-23 23:35:39,709 - DEBUG: come back to mainIE waitBusy after setForeground: 
2013-06-23 23:35:39,711 - DEBUG: self.mainIE.scrollToNode
2013-06-23 23:35:45,285 - DEBUG: waitBusy: True
2013-06-23 23:35:52,723 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 302, in <module>
    hasUnvisit = view_3_baobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 264, in view_3_baobei
    baobei.openCurBaobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 109, in openCurBaobei
    self.mainIE.scrollToNode(subNode)
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 245, in scrollToNode
    self.getWindow().scrollBy(0,delta*scrollDirection)
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 214, in getWindow
    return self.ie.Document.parentWindow
  File "C:\Python27\lib\site-packages\win32com\client\dynamic.py", line 511, in __getattr__
    ret = self._oleobj_.Invoke(retEntry.dispid,0,invoke_type,1)
com_error: (-2147352567, '\xb7\xa2\xc9\xfa\xd2\xe2\xcd\xe2\xa1\xa3', (0, None, None, None, 0, -2147467259), None)

2013-06-23 23:35:52,739 - DEBUG: len(ies): 3
2013-06-23 23:35:52,739 - DEBUG: ie.LocationURL: file:///G:/WorkStation/GitAutoVerify
2013-06-23 23:35:52,739 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop
2013-06-23 23:35:52,753 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.2.KpoEeQ&id=25283284088&bucket_id=19&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop&scene=taobao_shop
2013-06-23 23:35:53,767 - DEBUG: ie.LocationURL: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop
2013-06-23 23:35:53,767 - DEBUG: closeAllRunningIE: http://detail.tmall.com/item.htm?spm=a220o.1000855.w3-18405562880.4.KpoEeQ&id=17271840674&scene=taobao_shop
2013-06-24 00:03:56,154 - DEBUG: ===============================================Begin Log===============================================
2013-06-24 00:03:56,154 - DEBUG: 


2013-06-24 00:03:56,154 - DEBUG: batIdx: 0
2013-06-24 00:03:56,154 - DEBUG: numUnvisit: 6, self.numBaobei: 3
2013-06-24 00:03:56,154 - DEBUG: self.randomVisit: [3, 4, 0]
2013-06-24 00:03:56,233 - ERROR: Traceback (most recent call last):
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 302, in <module>
    hasUnvisit = view_3_baobei()
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 245, in view_3_baobei
    viewer.createBaobei(idx)
  File "G:\WorkStation\GitAutoVerify\TaobaoViewer.py", line 195, in createBaobei
    ieExplorer.openURL(url)
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 133, in openURL
    self.newIE(url)
  File "G:\WorkStation\GitAutoVerify\IEExplorer.py", line 127, in newIE
    self.ie = win32com.client.Dispatch("InternetExplorer.Application")
  File "C:\Python27\lib\site-packages\win32com\client\__init__.py", line 95, in Dispatch
    dispatch, userName = dynamic._GetGoodDispatchAndUserName(dispatch,userName,clsctx)
  File "C:\Python27\lib\site-packages\win32com\client\dynamic.py", line 114, in _GetGoodDispatchAndUserName
    return (_GetGoodDispatch(IDispatch, clsctx), userName)
  File "C:\Python27\lib\site-packages\win32com\client\dynamic.py", line 91, in _GetGoodDispatch
    IDispatch = pythoncom.CoCreateInstance(IDispatch, None, clsctx, pythoncom.IID_IDispatch)
com_error: (-2147023706, '\xd2\xd1\xbe\xad\xbc\xc6\xbb\xae\xcf\xb5\xcd\xb3\xb9\xd8\xbb\xfa\xa1\xa3', None, None)

2013-06-24 00:03:56,233 - DEBUG: len(ies): 1
2013-06-24 00:03:56,247 - DEBUG: ie.LocationURL: file:///G:/WorkStation/GitAutoVerify
